src/components/editMeasure/measureDetails/measureMetadata/MeasureMetadata.tsx

```diff
@@ -44,134 +32,119 @@ export default function MeasureMetadata(props: MeasureMetadataProps) {
-                    id={`measure-${typeLower}`}
-                    autoComplete={`measure-${typeLower}`}
-                    onChange={formik.handleChange}
-                    value={formik.values.genericField}
-                    placeholder={`${measureMetadataType}`}
-                    data-testid={`measure${measureMetadataType}Input`}
-                    {...formik.getFieldProps("genericField")}
-                  />
-                )}
-                {!canEdit && formik.values.genericField}
-
-                {formik.touched.genericField && formik.errors.genericField ? (
-                  <ErrorText>{formik.errors.genericField}</ErrorText>
-                ) : null}
-              </FieldSeparator>
-            </FormFieldInner>
-          </FormField>
-        </div>
-      </FormContent>
+      <div className="content">
+        <h3>{header}</h3>
+        {measureMetadataType === "Steward" && (
+          <SubHeader>
+            This information will be displayed publicly so be careful what you
+            share.
+          </SubHeader>
+        )}
+        <label htmlFor={`measure-${typeLower}`}>{measureMetadataType}</label>
+        <textarea
+          readOnly={!canEdit}
+          name={`measure-${typeLower}`}
+          id={`measure-${typeLower}`}
+          autoComplete={`measure-${typeLower}`}
+          onChange={formik.handleChange}
+          value={formik.values.genericField}
+          placeholder={`${measureMetadataType}`}
+          data-testid={`measure${measureMetadataType}Input`}
+          {...formik.getFieldProps("genericField")}
+        />
+      </div>
       {canEdit && (
-        <FormButtons>
-          <ButtonWrapper>
-            <Button
-              className="qpp-c-button--cyan"
-              type="submit"
-              data-testid={`measure${measureMetadataType}Save`}
-              disabled={!(formik.isValid && formik.dirty)}
-              style={{ marginTop: 20 }}
-            >
-              Save
-            </Button>
-
-            <MessageDiv>
-              {success && success.includes(`${measureMetadataType}`) && (
-                <SuccessText
-                  data-testid={`measure${measureMetadataType}Success`}
-                >
-                  Measure {measureMetadataType} Information Saved Successfully
-                </SuccessText>
-              )}
-              {error && error.includes(`${measureMetadataType}`) && (
-                <ErrorText data-testid={`measure${measureMetadataType}Error`}>
-                  {error}
-                </ErrorText>
-              )}
-            </MessageDiv>
-          </ButtonWrapper>
-        </FormButtons>
+        <div className="form-actions">
+          <button
+            className="cancel-button"
+            data-testid="cancel-button"
+            disabled={!formik.dirty}
+            onClick={() => resetForm()}
+          >
+            Discard Changes
+          </button>
+          <Button
```

Comment:
Any particular reason of why we are using 2 different button components ?

Reply:
This isn't a component, just the html for a button. We don't currently have button variations that match this in the design-system since we were under the impression that the `new library` and `new measure` button were going to be unique